src/TestUtilities/TestingAggregates/ExceptionInducingCommand.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EventSauce\EventSourcing\TestUtilities\TestingAggregates;
+
+use EventSauce\EventSourcing\DummyAggregateRootId;
+
+class ExceptionInducingCommand implements DummyCommand
+{
+    private DummyAggregateRootId $aggregateRootId;
+
+    public function __construct(DummyAggregateRootId $aggregateRootId)
```

Comment:
This class uses a `DummyAggregateRootId` rather than `AggregateRootId`: hard to understand if it's a test asset, or a source file :thinking:

Reply:
It's a test asset

Reply:
I've added a @testAssert marker on all the dummies and stubs.